### DISCUSSION TITLE
Can not verify my student card

### DISCUSSION THREAD

**Discussion by solo-driven** [community/community#51879] [GitHub Education]
I am a student and have trouble with my application. Every time I submit, that system cannot correctly locate me or my university and says "The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote."

Would be glad for any support. 
Thank you

**Comment by samus-aran** [comment]
Hi @solo-driven 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍
